src/EasyBib/Doctrine/DoctrineResource.php

```
@@ -316,18 +293,15 @@ protected function getEntityFolders()
     }
 
     /**
-     * Get Proxy folders
+     * Get Proxy folder
      *
      * @return string
      */
     protected function getProxyFolder()
     {
-        /*$folders = array(
-            $this->rootPath . '/library/Doctrine/Proxy'
-        );
-        if (is_dir($this->modulePath . '/' . $this->config->proxy->folder)) {
-            $folders[] = $this->modulePath . '/' . $this->config->proxy->folder;
-        }*/
+        if (is_dir($this->rootPath . '/' . $this->config->proxy->folder)) {
+            return $this->rootPath . '/' . $this->config->proxy->folder;
+        }
         return $this->rootPath . '/library/Doctrine/Proxy';
```

I am not 100% sure if it would be smarter to throw an exception here instead of returning the default, keeping it bc of "backwards compatibility" for now.